airmozilla/manage/urls.py

```diff
@@ -209,6 +209,9 @@
     url(r'^picturegallery/(?P<id>\d+)/delete/$',
         views.picture_delete,
         name='picture_delete'),
+    url(r'^picturegallery/delete_all/$',
```

Comment:
Actually, can we change this to:
```
url(r'^picturegallery/(?P<id>\d+)/delete-all/$'
```
And change the view function to be:
```python
def picture_delete_all(request, id):
   event = get_object_or_404(Event, id=id)
```
That way, the system automatically checks that you don't accidentally call `POST /picturegallery/delete_all/` without a `?event=123`.